docs/rfcs/archival_storage_service.md

```diff
@@ -0,0 +1,79 @@
+# Archival Storage Service
+
+## Problem Statement
+
+In order to provide archivists with reliable long term digital storage of both digitised and "born-digital" assets we need a service that can: 
+
+- Ensure safe long term storage assets of digitised assets
+- Follow industry practises around file integrity and audit trails
+- Provide a scalable mechinsm for indetifying and retrieving content
+
+## Suggested Solution
+
+We propose to build a storage service based on AWS S3 and DynamoDb. 
+
+![archival storage service - page 1](https://user-images.githubusercontent.com/953792/39753255-0987d2fe-52b6-11e8-97ee-633ac09e1a9e.png)
+
+### Process flow
+
+- Work is uploaded in gzipped BagIt format to S3 to an "Ingest Bucket"
```

Comment:
What is BagIt format? A glossary might be helpful.

Reply:
There is a `Terminology` section. I hacked together something after reading the RFC RFC: https://tools.ietf.org/html/rfc7990.